storefront/app/helpers/spree/filters_helper.rb

```
@@ -48,23 +52,27 @@ def products_count_for_filter(name, value_id)
     end
 
     def filter_stock_count
       @filter_stock_count ||= begin
-        products_scope = products_for_filters_scope(except_filters: [:purchasable, :out_of_stock])
+        products = products_for_filters_scope(except_filters: [:purchasable, :out_of_stock])
 
-        {
-          in_stock: products_scope.in_stock.distinct.count,
-          out_of_stock: products_scope.out_of_stock.distinct.count
-        }
+        Rails.cache.fetch(["stock-count", products], expires_in: 1.day) do
```

RuboCop (1.75.5)

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols.

(Style/StringLiterals)